docs/products/flink/howto/ext-pg-cdc-jdbc-connector.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Create a PostgreSQL® CDC or JDBC connector-based Apache Flink®
+---
+
+Learn how to create a table in your Apache Flink® application to interact with an external PostgreSQL® database.
+
+You can choose between two methods:
+
+- **Change Data Capture (CDC)**: Process real-time changes from the PostgreSQL tables
+  using Debezium.
+- **JDBC**: Execute SQL queries directly on the PostgreSQL database.
+
+## Prerequisites
+
+- An active Aiven for Apache Flink service.
+- An external PostgreSQL database with access details.
+- [Integration](/docs/products/flink/howto/ext-pg-flink-integration) between
+  Aiven for Apache Flink and external PostgreSQL.
+
+## Configure external PostgreSQL connector
+
+Follow these steps to configure an external PostgreSQL connector in your Aiven for Apache Flink
+application table, using either CDC or JDBC:
+
+1. Go to the **Aiven for Apache Flink** service page and
+   click **Application** in the sidebar.
+1. Click **Create new application**, enter a name your application, and
+   click **Create application**.
+
+   :::note
+    If modifying an existing application, create a new version for source or s
+    ink table changes.
+   :::
```

Comment:
I'm not sure I understand this note, what do we mean? also the `s <new link> ink` seems off 🙂

Reply:
Update text for clarity.